recipes/aws-c-event-stream/all/conanfile.py

```diff
@@ -39,13 +40,12 @@ def configure(self):
         del self.settings.compiler.libcxx
 
     def requirements(self):
-        self.requires("aws-c-common/0.4.25")
-        self.requires("aws-checksums/0.1.5")
+        self.requires("aws-checksums/0.1.11")
+        self.requires("aws-c-io/0.10.5")
```

Comment:
Is this a new dependency?
Is it version dependent?

Reply:
Yes. Shall I write a switch to manage the dependency?

Reply:
aws-c-io dependency was added here: https://github.com/awslabs/aws-c-event-stream/commit/8a98dec0 (before version 0.2.2)